test/Core.Test/Services/OrganizationServiceTests.cs

```diff
@@ -1685,25 +1685,6 @@ public async Task HasConfirmedOwnersExcept_WithConfirmedProviderUser_IncludeProv
         Assert.Equal(includeProvider, result);
     }
 
-    [Theory]
```

Comment:
We should add more test to check that this plans available and also works for secrets manager